.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1410,6 +1409,27 @@ TODO(lxea): replace links with actual docs once merged
   - [ ] Windows Desktop
   - [ ] App Access
 
+## SSH Connection Resumption
+
+Verify that SSH works, and that resumable SSH is not interrupted across a Teleport Cloud tenant upgrade. 
+|   | Standard node | Non-resuming node | Peered node |
+|---|---|---|---|
+| `tsh ssh` | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| `tsh ssh --no-resume` | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| Teleport Connect | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| Web UI (not resuming) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| OpenSSH (standard `tsh config`) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| OpenSSH (adding `--no-resume`) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
```

Comment:
I find this a bit ambiguous. I assume this means using an openssh client with `--no-resume` in the `ProxyCommand`, but if I were to perform this procedure I wouldn't be too sure about it.  Might be worth clarifying.